test/cypress/integration/group-chat-scrolling.spec.js

```diff
@@ -85,23 +85,17 @@ describe('Send/edit/remove messages & add/remove emoticons inside group chat', (
     cy.giLogout()
   })
 
-  it(`user2 joins ${groupName1} group and sends another 15 messages and reply a message`, () => {
+  it(`user2 joins ${groupName} group and sends another 15 messages and reply a message`, () => {
     cy.giAcceptGroupInvite(invitationLinkAnyone, {
       username: user2,
-      groupName: groupName1,
+      existingMemberUsername: user1,
+      groupName: groupName,
       shouldLogoutAfter: false,
       bypassUI: true
     })
     me = user2
     cy.giRedirectToGroupChat()
 
-    cy.getByDT('channelName').should('contain', CHATROOM_GENERAL_NAME)
-    cy.giCheckIfJoinedChatroom(CHATROOM_GENERAL_NAME, me)
-
-    cy.getByDT('channelsList').find('ul>li:first-child').within(() => {
-      cy.get('[data-test]').should('contain', CHATROOM_GENERAL_NAME)
-    })
-
```

Comment:
That's because we do this in `giAcceptGroupInvite`.
When it's the first time login, we check if they've fully synced general chatroom. It is defined in `giLogin` command especially line 123~125 in commands.js.